wsclient/src/main/java/org/hpccsystems/ws/client/utils/Utils.java

```diff
@@ -64,14 +68,24 @@ public static String parseVersionFromWSDLURL(String wsdlurl)
 
     static public void println(PrintStream stream, String message, boolean onlyifverbose, boolean verbosemode )
     {
-        if (verbosemode || !onlyifverbose)
-            stream.println(message);
+        // if (verbosemode || !onlyifverbose)
+        //     stream.println(message);
+        if (onlyifverbose) {
```

Comment:
if (onlyverbose && !verbosemode)

Reply:
@rpastrana Does using the incoming verbosemode still make sense if we are using log4j? Wouldn't we want the user to control that strictly via the log4j settings?

Reply:
I think so, its still a parameter in the methid signature, it'd be odd to ignore it. Unless targeting debug vs error in log4j implies verbose va nonverbose?